### PR TITLE
Show error message in AddSourceWizard

### PR DESCRIPTION
**Pull request by rvsia**
**Description**

- Shows error message in the error step:

image

- Tries to remove the source, if something goes wrong, 
       - if the deletion is unsuccessful, there is a message telling user to try to remove it again later

TODO:
- [x] tests

@Hyperkid123 @karelhala @terezanovotna

### CODE REVIEW

File: packages/sources/src/api/handleError.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { getSourcesApi } from './index';
+
+export const handleError = (id, { config: { url }, response: { data: { errors: [{ detail }] } } }) => getSourcesApi()
+.deleteSource(id)
+.catch((errorDelete) => {
+    throw `${url}: ${detail}. The source was not removed, try remove it later: ${errorDelete.response.data.errors[0].detail}`; }
+)
+.then(() => {
+    throw `${url}: ${detail}`;
```
Comment (Hyperkid123) [inline packages/sources/src/api/handleError.js]: I suppose you want to throw multiple errors?

Comment (rvsia) [reply packages/sources/src/api/handleError.js]: `Promise.All `returns only the first error, so no, I just want to show the first returned error.

Comment (Hyperkid123) [reply packages/sources/src/api/handleError.js]: Yeah i know i am just curious why there is throw in `then` branch.

Comment (rvsia) [reply packages/sources/src/api/handleError.js]: This is like an error 'post-process', the error is then catched by the outer catch branch.

Comment (Hyperkid123) [reply packages/sources/src/api/handleError.js]: You can do mutations via graphql? I though its not supported.

Comment (rvsia) [reply packages/sources/src/api/handleError.js]: I don't understand what you mean

**Comment by rvsia** [comment]
@Hyperkid123 updated! :+1:

**Pull request merged**